flocker/control/_config.py

```diff
@@ -1296,7 +1296,7 @@ def deployment_from_configuration(deployment_state, deployment_configuration,
             raise ConfigurationError("No known node with address {}.".format(
                 hostname))
         node = Node(uuid=node_states[hostname].uuid,
-                    applications=frozenset(node_applications),
+                    applications={app.name: app for app in node_applications},
```

Comment:
Yea I think we should probably remove it. I think it just gets obviously nuked in a move to something like Flocker-Lite.